index.js

```diff
@@ -72,22 +80,34 @@ var pluralTypes = {
     if (n === 1) { return 0; }
     return n % 10 >= 2 && n % 10 <= 4 && (n % 100 < 10 || n % 100 >= 20) ? 1 : 2;
   },
-  icelandic: function (n) { return (n % 10 !== 1 || n % 100 === 11) ? 1 : 0; }
+  icelandic: function (n) { return (n % 10 !== 1 || n % 100 === 11) ? 1 : 0; },
+  slovenian: function (n) {
+    if (n === 1 || n % 100 === 1) {
```

Comment:
This seems incorrect; it's numbers ending in *01* - so, `n === 1 || (n > 100 && n % 100 === 1)`

Reply:
I add the fix, but i don't beleive a number can be 01...since i'm assuming your passing in a number vs. a string.

Reply:
right - that's why the `n > 100`. because the only way a number can end in "01" is if it has three or more digits.